core/templates/dev/head/pages/collection_editor/collection_editor.html

```diff
@@ -121,7 +120,7 @@
 
   <script src="{{TEMPLATE_DIR_PREFIX}}/services/explorationServices.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/services/LocalStorageService.js"></script>
-
+  
```

Comment:
Drop extra spaces. If you're using sublime, you can add "trim_trailing_white_space_on_save": true, to your user prefs

Reply:
thanks @seanlip, didn't knew that!